S/Sundials/Sundials@5/build_tarballs.jl

```diff
@@ -66,6 +85,7 @@ fi
 # We attempt to build for all defined platforms
 platforms = filter!(p -> arch(p) != "powerpc64le", supported_platforms(; experimental=true))
 platforms = expand_gfortran_versions(platforms)
+push!(platforms, Platform("x86_64", "linux"; sanitize="memory"))
```

Comment:
Don't you need to do this before the libgfortran expansion?

Reply:
Since we're building with flang, it's using the flang fortran runtime anyway. We may want to reflect that in the fortran tag somehow.

Reply:
I see, right. I haven't checked this build yet (away from computer), but last time I had a look at flang my understanding was that the runtime is statically linked, so it shouldn't really need a tag of its own, right? Or you're worried about somehow mixing up code built with libgfortran and flang?

Reply:
It has a dynamically linked runtime and math library. For now, I think we can just leave it tagless though. I'm not sure classic flang will be our long-term solution here.

Reply:
Ok, the flang I looked at was probably a different one: https://twitter.com/owainkenway/status/1505326986243235841